README.md

```diff
@@ -4,6 +4,19 @@ The Concurrent Design Platform Java Software Development Kit that is compliant w
 
 > The C# SDK is the primary implementation, therefore the Java implementation may be a bit behind on new features and bug fixes. Both libraries are actively maintained and the Java implementation will get sycned up within a few weeks.
 
+# Maven Central
+
+The SDK contains multiple libraries that are each packaged as Maven artifacts and avaialble from [Maven Central](https://search.maven.org/).
```

Comment:
I would use MVNRepository links, as they are used the most by Java developers.
(https://mvnrepository.com/artifact/com.rheagroup)